  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 10, in <module>
    quarters, name_dict = pbp_parser(url)
  File "C:\Users\surma\Python\cfb_slapPY\pbp_parser_func.py", line 14, in pbp_parser
    html = urlopen(url, context=ctx).read()
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 525, in open
    response = meth(req, response)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 634, in http_response
    response = self.parent.error(
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 557, in error
    result = self._call_chain(*args)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 496, in _call_chain
    result = func(*args)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 749, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 525, in open
    response = meth(req, response)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 634, in http_response
    response = self.parent.error(
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 563, in error
    return self._call_chain(*args)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 496, in _call_chain
    result = func(*args)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "C:\Users\surma\Python\cfb_slapPY\sauce.py", line 13, in <module>
    quarters, name_dict = taster(url)  
  File "C:\Users\surma\Python\cfb_slapPY\taster_func.py", line 105, in taster
    abbr = abbr_finder(quarters[0][drive_count],abbr)
IndexError: list index out of range
Final score LaGrange 21 - Berry 34
Final score Southern Virginia 34 - LaGrange 38
Score correction made in N.C. Wesleyan @ LaGrange (index) quarter 0 drive 3
Final score LaGrange 21 - N.C. Wesleyan 26
Player name overlap detected in N.C. Wesleyan at LaGrange. Check possession transitions
Final score Methodist 34 - LaGrange 17
Final score LaGrange 10 - Brevard 20
Final score Maryville (Tenn.) 52 - LaGrange 14
Final score LaGrange 29 - Greensboro 14
Final score Averett 31 - LaGrange 22
Final score LaGrange 9 - Huntingdon 55
